scripts/motor_diagram_data/connect-escs-and-motors.rst

```diff
@@ -19,163 +19,15 @@ Motor order diagrams
 ====================
 
 The diagrams below show motor order for each frame type.
-The numbers indicate which output pin from the autopilot should be connected to each motor/propeller.
-The propeller direction is shown in green (clockwise, CW) or blue (counter-clockwise, CCW)
+The propeller direction is shown in green (clockwise, CW) or blue (counter-clockwise, CCW).
+The letters shown in red indicate which motor should turn when using Mission Planner's Motor Test feature.
 
 .. figure:: ../images/MOTORS_CW_CCWLegend.jpg
    :target: ../_images/MOTORS_CW_CCWLegend.jpg
 
    Legend for motor-order diagrams
 
-Quadcopter
-----------
-
-.. image:: ../images/motororder-quad-x-2d.png
-    :target: ../_images/motororder-quad-x-2d.png
-    :scale: 40%
-
-.. image:: ../images/motororder-betaflight-x-2d.png
-    :target: ../_images/motororder-betaflight-x-2d.png
-    :scale: 40%
-
-.. image:: ../images/motororder-bf-reversed-x-2d.png
-    :target: ../_images/motororder-bf-reversed-x-2d.png
-    :scale: 40%
-
-.. image:: ../images/motororder-dji-x-2d.png
-    :target: ../_images/motororder-dji-x-2d.png
-    :scale: 40%
-
-.. image:: ../images/motororder-quad-cw-x-2d.png
-    :target: ../_images/motororder-quad-cw-x-2d.png
-    :scale: 40%
-
-.. image:: ../images/motororder-quad-plus-2d.png
-    :target: ../_images/motororder-quad-plus-2d.png
-    :scale: 40%
-
-.. image:: ../images/motororder-quad-v-2d.png
-    :target: ../_images/motororder-quad-v-2d.png
-    :scale: 40%
-
-.. image:: ../images/motororder-quad-h-2d.png
-    :target: ../_images/motororder-quad-h-2d.png
-    :scale: 40%
-
-.. image:: ../images/motororder-quad-vtail-2d.png
-    :target: ../_images/motororder-quad-vtail-2d.png
-    :scale: 35%
-
-.. image:: ../images/motororder-quad-atail-2d.png
-    :target: ../_images/motororder-quad-atail-2d.png
-    :scale: 35%
-
-.. image:: ../../../images/motororder-Y4a-2d.jpg
-    :target: ../_images/motororder-Y4a-2d.jpg
-    :scale: 40%
-
-Tricopter
----------
-
-.. image:: ../images/motororder-tri-a-2d.png
-    :target: ../_images/motororder-tri-a-2d.png
-    :scale: 40%
-
-.. image:: ../images/motororder-tri-b-2d.png
-    :target: ../_images/motororder-tri-b-2d.png
-    :scale: 40%
-
-.. note::
```

Comment:
how does the diagram generator add this note?

Reply:
It lives in `scripts/motor_diagram_data/AP_Motors_display.json`